front/pages/w/[wId]/u/chat/index.tsx

```diff
@@ -252,37 +260,44 @@ export default function AppChat({
 
   return (
     <AppLayout user={user} owner={owner} gaTrackingId={gaTrackingId}>
-      <div className="flex flex-col">
-        <div className="mt-2 flex flex-initial">
+      <div className="flex h-full flex-col">
+        <div className="mt-2">
           <MainTab currentTab="Chat" owner={owner} />
         </div>
-        <div className="">
-          <div className="mx-auto mt-8 max-w-2xl px-6">
-            <div className="text-sm">
-              {messages.map((m, i) => {
-                return (
-                  <div key={i}>
-                    <MessageView
-                      user={user}
-                      message={m}
-                      loading={false}
-                    ></MessageView>
-                  </div>
-                );
-              })}
-              {response ? (
-                <div key={messages.length}>
-                  <MessageView
-                    user={user}
-                    message={response}
-                    loading={true}
-                  ></MessageView>
+        <div className="flex-1">
+          <div
+            className="h-full max-h-full grow-0 overflow-y-auto"
+            ref={scrollRef}
+          >
+            <div className="max-h-0">
```

Comment:
This is the only way I found to trigger the scrollable without the parent div growing itself.

Basically we overflow that div which don't grow the parent div but since there is content, it scrolls.